### PR TITLE
[#63]: Add campaign dates.

### PR DESCRIPTION
**Pull request by tjgupta**

### CODE REVIEW

File: app/models/campaign.js [hunk 1]
```diff
@@ -10,6 +10,13 @@ var CampaignSchema = new Schema({
   learn_more: {type: String, trim: true},
   publish: {type: Boolean, trim: true},
   influencer: {type: String, trim: false}
+}, {
```
Comment (Dfenestrator) [inline app/models/campaign.js]: Nit: indent

File: app/models/campaign.js [hunk 2]
```diff
@@ -10,6 +10,13 @@ var CampaignSchema = new Schema({
...
+  timestamps: {
+    createdAt: 'created_at',
+    updatedAt: 'updated_at'
+  }
 });
 
-mongoose.model('Campaign', CampaignSchema);
+var Campaign = mongoose.model('Campaign', CampaignSchema);
```
Comment (Dfenestrator) [inline app/models/campaign.js]: Do you know what the difference is here? what does model.exports do vs not calling it?

File: app/views/campaigns.hbs [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <div ng-controller="campaignController" >
     	<div ng-repeat="campaign in campaigns" class="background-white">
             <div class="campaign-info">
-                <h5>{[{campaign.date}]}</h5>
+                <h5 class="campaign-date">{[{campaign.created_at | date: 'shortDate'}]}</h5>
```
Comment (Dfenestrator) [inline app/views/campaigns.hbs]: Can you change this to the full date (March 7, 2017) for now?

File: public/css/campaigns.css [hunk 4]
```diff
@@ -42,6 +42,12 @@ h3, h4, h5, p {
     max-width: 100vw;
 }
 
+.campaign-date {
```
Comment (Dfenestrator) [inline public/css/campaigns.css]: Looks OK for now, but let's review with design to finalize. (we can merge for now and have Hendrika look at it)

File: app/models/campaign.js (hunk 2 above)
Comment (tjgupta) [inline app/models/campaign.js]: There's not really a difference at this point (in this PR). `module.exports` is just a way to expose the "API" of the module to other modules. It was mostly in preparation for a future PR. I can pull it out of this one if you prefer and add it to the next one.

File: app/views/campaigns.hbs (hunk 3 above)
Comment (tjgupta) [inline app/views/campaigns.hbs]: Yep, will do.

**Comment by tjgupta** [comment]
Fixed mentioned issues.

File: app/models/campaign.js (hunk 2 above)
Comment (Dfenestrator) [inline app/models/campaign.js]: No need, was just asking :)

**Pull request merged**